test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+odoo14-addon-partner-tz @ git+https://github.com/trobz/partner-contact@14.0-mig-partner_tz#subdirectory=setup/partner_tz
+odoo14-addon-base-time-window @ git+https://github.com/trobz/server-tools@14.0-mig-base_time_window#subdirectory=setup/base_time_window
+odoo14-addon-stock-partner-delivery-window @ git+https://github.com/trobz/stock-logistics-workflow@14.0-mig-stock_partner_delivery_window#subdirectory=setup/stock_partner_delivery_window
```

Comment:
Dependencies have been merged:
- https://github.com/OCA/edi/pull/332
- https://github.com/OCA/server-tools/pull/1988
- https://github.com/OCA/stock-logistics-workflow/pull/775

Reply:
thx, I updated it

Reply:
My bad, we were missing `partner_identification_import` when I commented, but now it is merged, you can remove this file now :)

Reply:
Hi @sebalix , I misunderstood too :(. I just pushed the update